notebooks/galaxy_redshift/redshift_fitting.ipynb

```diff
@@ -0,0 +1,619 @@
+{
```

Comment:
<p>Thank you so much for updating the notebook! I was able to easily follow along with the tutorial thanks to the figures you included. I do have one suggestion though - have you considered centering the figures? It's totally up to you whether or not you want to make that change, just thought I'd offer the idea. :) </p><pre class="ql-syntax" spellcheck="false">Now we need to:
- Open the ["line list" plugin](https://jdaviz.readthedocs.io/en/latest/specviz/plugins.html?highlight=plugin#line-lists)
&lt;center&gt;&lt;img src='./line_list.png' alt="The line list plugin is in the menu to the right" width="500"/&gt;&lt;/center&gt;


- Choose pre-loaded lines or add custom lines (the lines will not show in the viewer because they are plotted at restframe)
&nbsp; &nbsp; - Hint: select Ha and the surrounding NII lines
&lt;center&gt;&lt;img src='./select_list.png' alt="Select the SDSS list" width="500"/&gt;
&lt;img src='./select_three_lines.png' alt="Select the two [NII] lines and the H alpha line" width="500"/&gt;&lt;/center&gt;


- Input a guess redshift
&lt;center&gt;&lt;img src='./guess_redshift.png' alt="Input a guess for the redshift (redshift=2.4)" width="500"/&gt;&lt;/center&gt;


- Move the slider to get a better match
&lt;center&gt;&lt;img src='./adjust_redshift_slider.png' alt="Adjust the redshift using the slider to match the emission lines in the spectrum" width="500"/&gt;&lt;/center&gt;


- Use the [zoom tool](https://jdaviz.readthedocs.io/en/latest/specviz/displaying.html#pan-zoom) to get an even better match
&lt;center&gt;&lt;img src='./precise_adjust_with_zoom.png' alt="Use the zoom tool to find a better match to the emission lines" width="500"/&gt;&lt;/center&gt;
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Fun fact...the figures rendered centered on my machine, but rendered on the side online...not sure what to do about this! I kind of like them on the side though :)